README.md

```diff
@@ -28,24 +28,24 @@ A Police Computer Plugin and Server for LSPDFR.
 ## Citation and Arrest Options
 
 - Files: `EPC/citationOptions.json` & `EPC/arrestOptions.json`
-- `minFine`: Minimum fine in $ for charge
-- `maxFine`: Maximum fine in $ for charge
-- `minMonth`: Minimum jail time in month for charge (arrests only)
-- `maxMonth`: Maximum jail time in month for charge (arrests only), `null` for life in prison
+- `minFine`: Minimum fine in $ for a charge
```

Comment:
It might be clearer to use the charge instead of a charge

Reply:
gotcha